app/router.js

```diff
@@ -6,7 +6,7 @@ var Router = Ember.Router.extend({
 });
 
 export default Router.map(function() {
-  this.resource('matches', function() {
+  this.route('matches', { resetNamespace: true }, function() {
```

Comment:
@tauil this should not be in this PR, should be another one. It was a mistake. It fixes a deprecation waning that the Ember Inspector alert. Whant to remove from  this PR and open a new one?